crossbar/router/auth/pending.py

```diff
@@ -28,12 +28,39 @@
 #
 #####################################################################################
 
+import abc
 from autobahn.wamp import types
 from autobahn.wamp.exception import ApplicationError
 
 __all__ = ('PendingAuth',)
 
 
+class IRealmContainer(abc.ABC):
```

Comment:
this is great! it clearly defines the interface the proxy needs in the code for the incoming connection side to check for existence of realm-role requested by the client in the backend - which can be the router or poxy controller. cool, make sense